modules/navier_stokes/src/postprocessors/InternalVolumetricFlowRate.C

```diff
@@ -84,7 +85,7 @@ InternalVolumetricFlowRate::computeQpIntegral()
 
     const Elem * neighbor = _current_elem->neighbor_ptr(_current_side);
 
-    /// Get face value for velocity
+    // Get face value for velocity
     // FIXME Make sure getInternalFaceValue uses the right interpolation method, see #16585
     const auto & vx_face =
         !getFieldVar("vel_x", 0)
```

Comment:
I want to make sure that we end up in a place where we don't have any `getFieldVar` calls in `*::computeQpIntegral`. I think here (and in a lot of other places), we can actually just do `_fv_vel_x ? do_if_non_null : do_if_null`

Reply:
(obviously replacing `_fv_vel_x` with the appropriate variable in other contexts)